File: app/src/main/java/net/mbonnin/arcanetracker/Typefaces.java
package net.mbonnin.arcanetracker;

import android.graphics.Typeface;
import android.support.v4.content.res.ResourcesCompat;

import java.util.HashMap;
import java.util.Map;

import timber.log.Timber;

public class Typefaces {
    public static Typeface belwe() {
        return ResourcesCompat.getFont(ArcaneTrackerApplication.getContext(), R.font.belwe_bold);
    }

    public static Typeface franklin() {
        return ResourcesCompat.getFont(ArcaneTrackerApplication.getContext(), R.font.franklin_gothic);
    }
}

File: app/src/main/java/net/mbonnin/arcanetracker/Typefaces.kt
package net.mbonnin.arcanetracker

import android.graphics.Typeface
import android.support.v4.content.res.ResourcesCompat

object Typefaces {
    fun belwe(): Typeface? {
        return ResourcesCompat.getFont(ArcaneTrackerApplication.context, R.font.belwe_bold)
    }

    fun franklin(): Typeface? {
        return ResourcesCompat.getFont(ArcaneTrackerApplication.context, R.font.franklin_gothic)
    }
}
